Assets/MixedRealityToolkit/InputSystem/MixedRealityInputManager.cs

```diff
@@ -428,10 +430,10 @@ public IMixedRealityInputSource RequestNewGenericInputSource(string name, IMixed
         #region Input Source State Events
 
         /// <inheritdoc />
-        public void RaiseSourceDetected(IMixedRealityInputSource source)
+        public void RaiseSourceDetected(IMixedRealityInputSource source, IMixedRealityController controller = null)
```

Comment:
When would null be valid for controller? (in both Detected and Lost events)

Reply:
Yes for input sources that are not tied to a controller.

Reply:
Do we get detected | lost events in those cases?